### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by subnomo**
Updated pymesync requirement and added tests

### CODE REVIEW

**Comment by subnomo** [comment]
I removed nose in favor of a shell script for testing, but now I'm thinking that perhaps that wasn't a good idea.

**Comment by Kennric** [comment]
nose is a pretty good test runner, I think it will make life easier as the app gets more complex - add that back in and I will approve these tests

**Comment by subnomo** [comment]
@Kennric Done.

**Comment by Kennric** [comment]
+1

**Comment by Kennric** [comment]
I'd like to have one more review on this, @matthewrsj or @ElijahCaine ?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
 # timesync-frontend-flask
 
 Front-end webapp for the TimeSync API.
+
+To install requirements, do:
+```sh
+$ virtualenv venv
```
Comment (pop) [inline README.md]: We should add the -p <python version> flag if this only works on python 2 or 3.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+$ source venv/bin/activate
+(venv) $ pip install -r requirements.txt
+```
+
+To run tests, do:
+```
+(venv) $ nosetests
```
Comment (ElijahCaine) [inline README.md]: `nose` isn't actually included in the `requirements.txt`, so this doesn't work. Should it be?

We might be able to set up multiple requirement files for different environment (`dev.txt` includes `nose`, `prod.txt` only includes what you need to run the code, etc), but the setup guide should work out of the box.

**Comment by pop** [comment]
Besides the explicit python versioning in the setup README, everything looks good. Great work +1

**Pull request merged**